tests/src/python/test_qgsserver.py

```diff
@@ -475,6 +477,23 @@ def test_wms_getmap_basic(self):
         r, h = self._result(self.server.handleRequest(qs))
         self._img_diff_error(r, h, "WMS_GetMap_Basic")
 
+        qs = "&".join(["%s=%s" % i for i in list({
```

Comment:
Yes I totally agree with you! 

However, I think it's not the place to focus on this aspect. But I can propose another PR soon about it.